bindgen/ir/comp.rs

```diff
@@ -644,7 +659,11 @@ where
             // intentional.  Alignment of the allocation unit is based on the
             // maximum bitfield width, not (directly) on the bitfields' types'
             // alignment.
-            unit_align = cmp::max(unit_align, bitfield_width);
+            if bitfields_bit_begin_off == 0 {
+                unit_align = cmp::max(unit_align, bitfield_width);
+            } else {
+                unit_align = 8;
```

Comment:
This 8 look a bit magic, where does it come from? Is it from the pointer width? If so, it should use that, rather than hardcoding 8.

Reply:
`8` come from bit size of byte